dualis_connector/dualis_service.py

```diff
@@ -27,8 +33,8 @@ def interactively_acquire_token(self) -> str:
         token = None
```

Comment:
The print and dochl comment should be in the method above.

Also please add a doc comment to this method.

Reply:
Done